docs/experimental/viz_points_kde.py

```diff
@@ -0,0 +1,356 @@
+import numpy as np
+from fury import window, actor
+from fury.shaders import compose_shader, shader_apply_effects
+from fury.lib import Texture, WindowToImageFilter
+from fury.io import load_image
+from fury.utils import rgb_to_vtk
+from matplotlib import colormaps
+
+
+def window_to_texture(
+        window : window.RenderWindow,
+        texture_name : str,
+        target_actor : actor.Actor,
+        blending_mode : str = "None",
+        wrap_mode : str = "ClampToBorder",
+        border_color : tuple = (
+            0.0,
+            0.0,
+            0.0,
+            1.0),
+        interpolate : bool = True):
+    """Captures a rendered window and pass it as a texture to the given actor.
+
+    Parameters
+    ----------
+    window : window.RenderWindow
+        Window to be captured.
+    texture_name : str
+        Name of the texture to be passed to the actor.
+    target_actor : actor.Actor
+        Target actor to receive the texture.
+    blending_mode : str, optional
+        Texture blending mode. The options are:
+    1. None
+    2. Replace
+    3. Modulate
+    4. Add
+    5. AddSigned
+    6. Interpolate
+    7. Subtract
+
+    wrap_mode : str, optional
+        Texture wrapping mode. The options are:
+    1. ClampToEdge
+    2. Repeat
+    3. MirroredRepeat
+    4. ClampToBorder
+
+    border_color : tuple (4, ), optional
+        Texture RGBA border color.
+    interpolate : bool, optional
+        Texture interpolation."""
+
+    wrap_mode_dic = {"ClampToEdge" : Texture.ClampToEdge,
+                     "Repeat" : Texture.Repeat,
+                     "MirroredRepeat" : Texture.MirroredRepeat,
+                     "ClampToBorder" : Texture.ClampToBorder}
+
+    blending_mode_dic = {"None" : 0, "Replace" : 1,
+                         "Modulate" : 2, "Add" : 3,
+                         "AddSigned" : 4, "Interpolate" : 5,
+                         "Subtract" : 6}
+
+    r, g, b, a = border_color
+
+    windowToImageFilter = WindowToImageFilter()
+    windowToImageFilter.SetInput(window)
+
+    windowToImageFilter.Update()
+
+    texture = Texture()
+    texture.SetInputConnection(windowToImageFilter.GetOutputPort())
+    texture.SetBorderColor(r, g, b, a)
+    texture.SetWrap(wrap_mode_dic[wrap_mode])
+    texture.SetInterpolate(interpolate)
+    texture.MipmapOn()
+    texture.SetBlendingMode(blending_mode_dic[blending_mode])
+
+    target_actor.GetProperty().SetTexture(texture_name, texture)
+
+
+def texture_to_actor(
```

Comment:
You are not using this function

Reply:
Indeed, I forgot to take it out, thanks Tania